ui/src/variables/selectors/index.tsx

```diff
@@ -19,19 +25,19 @@ const getVariablesForOrgMemoized = memoizeOne(
   }
 )
 
-export const getVariablesForOrg = (state: AppState, orgID: string) => {
+export const getVariablesForOrg = (
+  state: AppState,
+  orgID: string
+): Variable[] => {
   return getVariablesForOrgMemoized(state.variables.variables, orgID)
 }
 
 export const getValueSelections = (
   state: AppState,
   contextID: string
 ): ValueSelections => {
-  const contextValues: VariableValues = get(
-    state,
-    `variables.values.${contextID}.values`,
-    {}
-  )
+  const contextValues: VariableValues =
+    get(state, `variables.values.${contextID}.values`) || {}
```

Comment:
does this sometimes have the key but it is set to null?

Reply:
Yep